backend/app/services/webhook_notification.py

```
@@ -0,0 +1,174 @@
+                    timeout=self.timeout
+                )
+            else:
+                logger.error(f"Unsupported HTTP method: {self.http_method}")
+                return False
+
+            response.raise_for_status()
+            logger.info(f"Webhook notification sent successfully for {notification.event} id={notification.id}")
+            return True
+
+        except httpx.HTTPError as e:
+            logger.error(f"HTTP error sending webhook notification: {str(e)}")
```

Ruff (0.14.5)

Use `logging.exception` instead of `logging.error`

Replace with `exception`

(TRY400)